src/analysis/raster/qgsrastercalcnode.h

```diff
@@ -125,6 +125,19 @@ class ANALYSIS_EXPORT QgsRasterCalcNode
 
     static QgsRasterCalcNode *parseRasterCalcString( const QString &str, QString &parserErrorMsg ) SIP_FACTORY;
 
+    /**
+     * Returns a List of Strings with raster name
```

Comment:
can you please improve , for example "Returns a list of raster layer names that are referenced in the formula" or something like that.